clpy/core/include/cl_stub.hpp

```diff
@@ -176,4 +178,5 @@ typedef enum{
 }cl_mem_fence_flags;
 __attribute__((annotate("clpy_no_mangle"))) void barrier(cl_mem_fence_flags);
 __attribute__((annotate("clpy_no_mangle"))) int printf(char const* format, ...);
+static constexpr double M_PI = 3.14;
```

Comment:
too bad...

Reply:
We must write an initial value when using `constexpr` , so I will leave some comments.